### PR TITLE
fix(IUI_Version3): fixed issues like ReactSelect, login, react-pdf component

### PR DESCRIPTION
**Pull request by Ojaghi1991**
Please ensure your pull request adheres to the following guidelines:

- [ ] Use the following format: `* [owner/repo](link)`
- [ ] Link additions should be added to the bottom of the relevant category.
- [ ] New categories or improvements to the existing categorization are welcome.
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] Sort by alphabetical order

Thanks for contributing!

### CODE REVIEW

File: components/form/reactSelect/reactSelect.js [hunk 1]
```diff
@@ -58,8 +58,14 @@ const ReactSelect = ({
         control={control}
         defaultValue={defaultValue}
         name={name}
-        render={(props) => (
-          <Component options={options} selectedOption={props.value} {...props} {...parentProps} />
+        render={({ field: { onChange, ...props } }) => (
+          <Component
+            options={options}
+            selectedOption={props.value}
+            onChange={onChange}
```
Comment (iwarner) [inline components/form/reactSelect/reactSelect.js:64]: @Ojaghi1991 Alpha

File: components/index.js [hunk 2]
```diff
@@ -150,23 +150,23 @@ export { default as MarkerClusterer } from './molecules/map/dynamicLocation/clus
 export { default as InfoWindow } from './molecules/map/dynamicLocation/infoWindow'
 export { default as StaticLocation } from './molecules/map/staticLocation/staticLocation'
 export { default as Navbar } from './molecules/navbar/navbar'
-export { default as PieChart } from './molecules/charts/nivo/pieChart/pieChart'
+// export { default as PieChart } from './molecules/charts/nivo/pieChart/pieChart'
```
Comment (iwarner) [inline components/index.js]: @Ojaghi1991 No commments

File: components/molecules/tile/tile.js [hunk 3]
```diff
@@ -112,13 +112,15 @@ Tile.propTypes = {
   loading: bool,
   rounded: bool,
   size: string,
-  title: string
+  title: string,
+  bgColour: string
```
Comment (iwarner) [inline components/molecules/tile/tile.js]: @Ojaghi1991 Alpha

**Review by iwarner — CHANGES_REQUESTED** [review]

File: components/index.js (hunk 2 above)
Comment (Ojaghi1991) [reply components/index.js]: @iwarner Do you mean remove PieChart? because there is no piechart in project

**Review by iwarner — APPROVED** [review]

**Pull request merged**